ConfigurationSystem/Client/Utilities.py

```diff
@@ -609,6 +613,8 @@ def getElasticDBParameters( fullname ):
       dbPassword = result['Value']
   else:
     dbPassword = result['Value']
-  parameters[ 'Password' ] = dbPassword
+  
+  if dbPassword:
+    parameters[ 'Password' ] = dbPassword
 
   return S_OK( parameters )
```

Comment:
Just double checking: you may end up that here you return parameters that contain no user, and/or no password. Is this what you want?